src/main/java/com/sesac/carematching/chat/config/WebSocketConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.sesac.carematching.chat.config;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.*;
+
+@RequiredArgsConstructor
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+
+    @Override
+    public void configureMessageBroker(MessageBrokerRegistry config) {
+        config.enableSimpleBroker("/topic", "/queue"); // 메시지 수신용
+        config.setApplicationDestinationPrefixes("/app"); // 메시지 발신용
+    }
+
+    @Override
+    public void registerStompEndpoints(StompEndpointRegistry registry) {
+        // CORS 설정을 통해 프론트엔드에서 접근 가능하게 함
+        registry.addEndpoint("/ws")
+            .setAllowedOrigins("http://localhost:3000") // 프론트엔드 도메인 허용
```

Comment:
```suggestion
            .setAllowedOrigins("https://d12hp6zm8su88f.cloudfront.net/",
                    "https://carematching.net/", "https://www.carematching.net/",
                    "http://localhost:3000", "http://localhost") // 프론트엔드 도메인 허용
```

저희 배포된 사이트에서도 CORS 허용되도록 주소 추가해주세요~